Homework1/BurrowsWheelerTransform/BurrowsWheelerTransform/StringTransformation.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+
+namespace BurrowsWheelerTransform
+{
+    public class StringTransformation
+    {
+        public string DirectBurrowsWheelerTransformation(string stringToConvert, ref int index)
```

Comment:
В index значение только присваивается, так что он должен быть out int index. Однако в C# года три как out-параметры не нужны, потому что он поддерживает кортежи:
```suggestion
        public (string, int) DirectBurrowsWheelerTransformation(string stringToConvert)
```